panel/panelpluginsmodel.cpp

```diff
@@ -246,9 +246,12 @@ QString PanelPluginsModel::findNewPluginSettingsGroup(const QString &pluginType)
     groups.sort();
 
     // Generate new section name
-    for (int i = 2; true; ++i)
-        if (!groups.contains(QStringLiteral("%1%2").arg(pluginType).arg(i)))
-            return QStringLiteral("%1%2").arg(pluginType).arg(i);
+    if (!groups.contains(QStringLiteral("%1").arg(pluginType)))
+        return QStringLiteral("%1").arg(pluginType);
+    else
+        for (int i = 2; true; ++i)
+            if (!groups.contains(QStringLiteral("%1%2").arg(pluginType).arg(i)))
```

Comment:
@rbazaud a small optimization would be to create the local `QString` as `QStringLiteral("%1%2").arg(pluginType).arg(i))` to not constuct it twice in case of return .... sure it's no real optimization as this is used only in case of adding new plugin, but IMO it would look cleaner